tabu/sampler.py

```diff
@@ -151,8 +151,12 @@ def sample(self, bqm, initial_states=None, initial_states_generator='random',
 
         # run Tabu search
         samples = np.empty((parsed.num_reads, len(bqm)), dtype=np.int8)
+
+        rng = np.random.RandomState(seed)
```

Comment:
I'd [switch to `default_rng`](https://numpy.org/doc/stable/reference/random/index.html#quick-start).